### ISSUE TITLE
Overlay: High Contrast Support

### ISSUE THREAD

**Issue by Boris-Em** [OfficeDev/office-ui-fabric-react#2858]
### Bug Report

- __Package version(s)__: 5.0
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Low

#### Describe the issue:

The overlay is not visible in high contrast dark mode.

#### Actual behavior:

![overlay](https://user-images.githubusercontent.com/4613643/30461727-6aa48f3c-9975-11e7-8b79-8201083e0f7f.png)
